kde-apps/arianna/arianna-23.08.5.recipe

```diff
@@ -0,0 +1,112 @@
+discovers your books automatically, and sorts them by categories, genres and authors."
+HOMEPAGE="https://invent.kde.org/graphics/arianna"
+COPYRIGHT="2022 Niccolò Venerandi"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/-/archive/v$portVersion/arianna-v$portVersion.tar.bz2"
+CHECKSUM_SHA256="cff07240a8aa25e8e0e3f298ca6f5369f090198a4b16712daba411980e06c06f"
+SOURCE_DIR="arianna-v$portVersion"
+PATCHES="arianna-$portVersion.patchset"
+ADDITIONAL_FILES="arianna.rdef.in"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	arianna$secondaryArchSuffix = $portVersion
+	cmd:arianna$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libKF5Archive$secondaryArchSuffix
+	lib:libKF5Baloo$secondaryArchSuffix
+	lib:libKF5ConfigCore$secondaryArchSuffix
+	lib:libKF5ConfigGui$secondaryArchSuffix
+	lib:libKF5CoreAddons$secondaryArchSuffix
+	lib:libKF5DbusAddons$secondaryArchSuffix
+	lib:libKF5FileMetaData$secondaryArchSuffix
+	lib:libKF5I18n$secondaryArchSuffix
+	lib:libKF5Kirigami2$secondaryArchSuffix
+	lib:libKF5WindowSystem$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5DBus$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5Sql$secondaryArchSuffix
+	lib:libQt5Svg$secondaryArchSuffix
+	lib:libQt5Webengine$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	lib:libQt5Xml$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	extra_cmake_modules$secondaryArchSuffix
+	kirigami_addons$secondaryArchSuffix
+	kquickcharts$secondaryArchSuffix
+	qqc2_desktop_style${secondaryArchSuffix}_devel
+	devel:libKF5Archive$secondaryArchSuffix
+	devel:libKF5Baloo$secondaryArchSuffix
+	devel:libKF5ConfigCore$secondaryArchSuffix
+	devel:libKF5CoreAddons$secondaryArchSuffix
+	devel:libKF5DbusAddons$secondaryArchSuffix
+	devel:libKF5FileMetaData$secondaryArchSuffix
+	devel:libKF5I18n$secondaryArchSuffix
+	devel:libKF5Kirigami2$secondaryArchSuffix
+	devel:libKF5WindowSystem$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Webengine$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:g++$secondaryArchSuffix
+	cmd:make
+	cmd:msgfmt
+	cmd:msgmerge
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:qdbuscpp2xml$secondaryArchSuffix >= 5
+	cmd:python3
+	"
+
+BUILD()
+{
+	cmake -Bbuild -S. $cmakeDirArgs \
+		-DCMAKE_BUILD_TYPE=Release \
+		$cmakeDirArgs \
+		-DECM_DIR=/system/data/cmake/Modules/ECM/cmake
+	make -Cbuild $jobArgs
+}
+
+INSTALL()
+{
+	make -Cbuild install
+
+	# we are not linux
+	rm -rf $dataDir/{applications,icons,metainfo}
+
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
+	local MINOR="`echo "$portVersion" | cut -d. -f3`"
+	local APP_NAME="Arianna"
+	local LONG_INFO="$SUMMARY"
+	local APP_SIGNATURE="application/x-vnd.mednaffe"
```

Comment:
I think this is wrong. Copy-paste error?

Reply:
That's what you get with templates ;) Will fix.

Reply:
Fixed in: https://github.com/haikuports/haikuports/pull/10337 Thanks!